prove_validator.go

```diff
@@ -33,55 +54,126 @@ func (epp *EigenPodProofs) ProveValidatorContainers(oracleBlockHeader *phase0.Be
 		return nil, err
 	}
 
-	VerifyValidatorFieldsCallParams := &VerifyValidatorFieldsCallParams{}
-	VerifyValidatorFieldsCallParams.StateRootProof = &StateRootProof{}
+	verifyValidatorFieldsCallParams := &VerifyValidatorFieldsCallParams{}
+
+	// Get the state root proof
+	verifyValidatorFieldsCallParams.StateRootProof = &StateRootProof{}
+	verifyValidatorFieldsCallParams.StateRootProof.BeaconStateRoot = oracleBlockHeader.StateRoot
+	verifyValidatorFieldsCallParams.StateRootProof.Proof, err = beacon.ProveStateRootAgainstBlockHeader(oracleBlockHeader)
+	if err != nil {
+		return nil, err
+	}
+
 	// Get beacon state top level roots
 	beaconStateTopLevelRoots, err := epp.ComputeBeaconStateTopLevelRoots(oracleBeaconState)
 	if err != nil {
 		return nil, err
 	}
 
-	// Get beacon state root.
-	VerifyValidatorFieldsCallParams.StateRootProof.BeaconStateRoot = oracleBlockHeader.StateRoot
+	verifyValidatorFieldsCallParams.OracleTimestamp, err = GetSlotTimestamp(oracleBeaconState, oracleBlockHeader)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
This is wrong in PEPE. Not sure how you want to correct it - the CLI doesn't actually use it but we should maybe make sure other shit isn't using it either?